bitnami/postgresql-repmgr/10/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -469,6 +538,7 @@ repmgr_generate_repmgr_config() {
     # set the "--waldir" option accordingly
     local -r waldir=$(postgresql_get_waldir)
     local -r waldir_option=$([[ -n "$waldir" ]] && echo "--waldir=$waldir")
+    local -r randomize=$(( ( RANDOM % 10 )  + 5 ))
```

Comment:
randomize is not needed, our libraries include the function `generate_random_string` which can be used to generate numeric values:
```
generate_random_string --type numeric --count 1
```